server/README.md

```diff
@@ -24,25 +24,40 @@ Before doing any of the commands below, make sure to install the Python packages
 1. In the first shell window, run `. ./run.sh --dev`
 1. The server should now be located at `localhost:5000`
 
+If an issue comes up where it says that the port is already used, check to see if redis is running in another terminal somewhere. Or it might have been installed as a service and running in the background, which means it does not need to be started again.
+
+Sometimes while developing it can be useful to clear the redis DB of data if issues arise with the models. This can be done in a separate terminal than that which is running redis by typing `redis-cli FLUSHDB`.
```

Comment:
Redis is usually run as a service so you don't need a terminal open. I know this is mentioned here but I think the language should treat that as the default thing to look at it. After all its pretty obvious if you have a terminal open that's running redis and that's whats causing the conflict. It's less likely that they'll pick up on a background service they might've had running for the last year unknowingly.

It's a little change but something I'd encourage.

Reply:
Changed it to the below:

If an issue comes up where it says that the port is already used, check to see if redis is running in another terminal somewhere, or in a background service. By default redis runs as a service, so it might not be necessary to run it in another terminal if it is.

Does that sound good or what should it be?

Can't remember how I installed it but it doesn't run as a service on my computer and it is started manually. Didn't constantly want it running in the background so I might have chosen to do that specifically.

Reply:
What about:

If an issue comes up where the port used by redis is already in use, you may already have it running through a background service or an open terminal. You can check to see if redis is already running with 'X' command. If it is, you're good to go and don't need to run it again. If it's not, you may have some other service or application active that may be making use of this port. If this is the case, here's a link to instructions on how to configure redis to use a different port: <link>.

Sometimes while developing it can be useful to clear the redis DB of data if issues arise with existing models. This can be done in any inactive terminal with the command 'redis-cli FLUSHDB'.